### PR TITLE
[keymap(kyria), refactor] created kyria keymap. refactored ninjonas userspace

### PR DESCRIPTION
**Pull request by ninjonas**
## Description
- Adding new kyria keymap
- Minor tweaks to ninjonas userspace

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/kyria/keymaps/ninjonas/rules.mk [hunk 1]
```diff
@@ -0,0 +1,5 @@
+OLED_DRIVER_ENABLE = yes   # Enables the use of OLED displays
+ENCODER_ENABLE = yes       # Enables the use of one or more encoders
+RGBLIGHT_ENABLE = yes      # Enable keyboard RGB underglow
+LINK_TIME_OPTIMIZATION_ENABLE = yes
+EXTRAFLAGS += -flto
```
Comment (Duckle29) [inline keyboards/kyria/keymaps/ninjonas/rules.mk]:
```suggestion
```

That's what `LINK_TIME_OPTIMIZATION_ENABLE = yes` does :)

**Comment by ninjonas** [comment]
> The change requested above ^

Thanks @Duckle29 for the code review. Just pushed the changes requested.

File: users/ninjonas/ninjonas.h [hunk 2]
```diff
@@ -62,13 +64,13 @@
 
 #define _____________________DVORAK_L1______________________ T_TAB, KC_QUOT, KC_COMM, KC_DOT, KC_P, KC_Y
 #define _____________________DVORAK_L2______________________ T_ESC, KC_A, KC_O, KC_E, KC_U, KC_I
-#define _____________________DVORAK_L3______________________ KC_LSFT, KC_SCLN, T_Q, KC_J, KC_K, KC_X
+#define _____________________DVORAK_L3______________________ KC_LSFT, KC_SCLN, KC_W, KC_J, KC_K, KC_X
```
Comment (noroadsleft) [inline users/ninjonas/ninjonas.h]:
```suggestion
#define _____________________DVORAK_L3______________________ KC_LSFT, KC_SCLN, KC_Q, KC_J, KC_K, KC_X
```

File: users/ninjonas/ninjonas.h [hunk 3]
```diff
@@ -62,13 +64,13 @@
...
 
 #define _____________________DVORAK_R1______________________ KC_F, KC_G, KC_C, KC_R, KC_L, KC_BSLS
 #define _____________________DVORAK_R2______________________ KC_D, KC_H, KC_T, KC_N, KC_S, KC_SLSH
-#define _____________________DVORAK_R3______________________ KC_B, KC_M, T_W, KC_V, KC_Z, MT_EQL
+#define _____________________DVORAK_R3______________________ KC_B, KC_M, KC_W, KC_V, KC_Z, MT_EQL
 
-#define _____________________COLEMAK_L1_____________________ T_TAB, T_Q, T_W, KC_F, KC_P, KC_G
+#define _____________________COLEMAK_L1_____________________ T_TAB, KC_W, KC_W, KC_F, KC_P, KC_G
```
Comment (noroadsleft) [inline users/ninjonas/ninjonas.h]:
```suggestion
#define _____________________COLEMAK_L1_____________________ T_TAB, KC_Q, KC_W, KC_F, KC_P, KC_G
```

File: users/ninjonas/ninjonas.h (hunk 2 above)
Comment (ninjonas) [reply users/ninjonas/ninjonas.h]: Doh! That explains a lot! Been trying to learn these layouts. Good catch, thank you!

File: users/ninjonas/ninjonas.h (hunk 3 above)
Comment (ninjonas) [reply users/ninjonas/ninjonas.h]: whoops!

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**